sdk/core/azure-core/py.typed

```diff
@@ -0,0 +1,27 @@
+# --------------------------------------------------------------------------
```

Comment:
You don't need this file, azure-core has already [the right file in the right place](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/azure/core/py.typed). I encourage you read PEP561 to understand better the role and placement of this file.